### PR TITLE
@W-15006946 - Release v2.14.0

### PR DESCRIPTION
**Pull request by alexvuong**
Release for adding new. Shopper SEO APIs

### CODE REVIEW

**Comment by alexvuong** [comment]
Note: I pushed forced because I made a mistake that having 2.13.1 version in the commits. I rebased locally and fixed up that commit.

File: apis/product/shopper-products/shopper-products.raml [hunk 1]
```diff
@@ -880,11 +875,9 @@ types:
             example: USD
           expand:
             description: The expand parameter. A comma separated list with the allowed values (availability, links, promotions, options, images, prices, variations, recommendations). All expand parameters are used for the request when no expand parameter is provided. The value "none" may be used to turn off all expand options.
-            type: array
```
[the remark this answers is not in the document]
Comment (alexvuong) [reply apis/product/shopper-products/shopper-products.raml]: @joeluong-sfcc I've reverted the change on the shopper products. Thanks for spotting that

**Review by joeluong-sfcc — COMMENTED** [review]
Apologies, I forgot that you'll also have to re-run the docs command since the docs now reflect the incorrect shopper products RAML:

Screenshot 2024-03-04 at 5 57 58 PM

`expand` should be type array and not string

**Review by joeluong-sfcc — APPROVED** [review]
LGTM, thanks for handling the release

**Review by bendvc — APPROVED** [review]

**Pull request merged**